app/src/main/java/eu/kanade/tachiyomi/source/online/HttpSource.kt

```
@@ -197,16 +197,21 @@ abstract class HttpSource : CatalogueSource {
 
     /**
      * Returns an observable with the updated chapter list for a manga. Normally it's not needed to
-     * override this method.
+     * override this method.  If a manga is licensed an empty chapter list observable is returned
      *
      * @param manga the manga to look for chapters.
      */
     override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
-        return client.newCall(chapterListRequest(manga))
-                .asObservableSuccess()
-                .map { response ->
-                    chapterListParse(response)
-                }
+        if (manga.status != SManga.LICENSED) {
+            return client.newCall(chapterListRequest(manga))
+                    .asObservableSuccess()
+                    .map { response ->
+                        chapterListParse(response)
+                    }
+        } else {
+            val chapters: List<SChapter> = emptyList()
+            return Observable.just(chapters)

```

Probably better to throw an exception? `Observable.error(Exception("Licensed"))` for example

---

`return Observable.error(Exception("Licensed - No chapters to show"))`

Added a little more detail